helpt/urls.py

```diff
@@ -17,6 +17,7 @@
 from django.contrib import admin
 from rest_framework.routers import DefaultRouter
 
+from projects.adapters.github_hook import urls as github_urls
```

Comment:
I think `github.py` has not grown to be too big to include the hook code, as well. So move the contents of `github_hook.py` into `github.py`.